### Respuestas a las preguntas 3 y 5 de examen

La posición a la que debería colocarse un satélite geoestacionario:

In [16]:
Me = 5.972e24  # Masa de la tierra (Kg)
G  = 6.672e-11 # Cte. de gravedad universal (N*m^2/kg^2)
T  = 24*3600   # Periodo del satélite en segundos
M_sat = 30e3   # Masa del satélite (Kg)

Re  = 6371e3  # Radio de la tierra (m)


Empleando la relación de la tercera ley de Kepler:
$$
  T^2 = \frac{4\pi^2*a^3}{G*M_e}
$$

donde a es la distancia del semieje mayor, considerando que es una órbita circular:
$$
  a = R_e + h
$$

$R_e$ es el radio medio de la tierra y $h$ es la altura del satélite.

In [17]:
import numpy as np
h = np.power(G*Me*np.power(T,2)/(4*np.power(np.pi,2)),1/3)-Re
print("La distancia para un satélite geoestacionario: {:.2f} kilometros".format(h/1000))

La distancia para un satélite geoestacionario: 35864.85 kilometros


&nbsp;






Un satélite cruza el ecuador de la tierra a las 12:00 pm en dirección de sur a norte. El satélite orbitá a una altura de 600 Km de la superficie, calcular el siguiente cruce para determinar su visibilidad desde el mismo punto.  

El satélite tendrá un periodo de:

In [18]:
h = 600e3
T_sat = 2*np.pi*np.sqrt(np.power(Re+h,3)/(G*Me)) # Despreciando la masa del satélite
print("{:.2f} segundos".format(T_sat))


5793.41 segundos


Esto es, la siguiente vez que cruzara el ecuador será en $T_{sat}/2 = 2896.7$ segundos, pero viajando de norte a sur. Por lo que el nuevo cruce de sur a norte será en $T_{sat} = 5793.41$ segundos.

La siguiente imagen esquematiza lo anterior:

<img src="./Imagenes/cruce.png" alt="esquema geometrico cruce" style="width: 600px;"/>

En una vista desde el eje polar, vista desde "arriba", el punto sobre el ecuador cuando sucede el cruce se traslada una distancia angular a una razón de 360°/86400seg

Por lo que en $T_{sat}$ habrá avanzado:

In [19]:
ang_avan = 360*T_sat/86400
print("Avance angular del punto en el ecuador en {:.2f} es: {:.2f} grados".format(T_sat,ang_avan))

Avance angular del punto en el ecuador en 5793.41 es: 24.14 grados


<img src="./Imagenes/satelite_polar.png" alt="esquema geometrico polar" style="width: 600px;"/>

Para determinar si el punto se encuentra en el área de cobertura, se calcula el ángulo $\theta$ del esquema geométrico del satélite para área de cobertura.

$$
  cos (\theta) = \frac{R_e}{R_e+h}
$$

In [20]:
ang_cover = np.degrees(np.arccos(Re/(Re+h)))
print("Angulo: {:.2f} grados".format(ang_cover))

Angulo: 23.95 grados


Esto es, el angulo $\theta$ del esquema de cobertura es menor al punto desplazado en el ecuador. Por tanto, no es visible el satélite en el siguiente paso.